packages/bitcore-node/docs/wallet-guide.md

```diff
@@ -0,0 +1,63 @@
+## Set up Alias config
+
+Go to the root directory of your computer
+
+```
+cd ~
+```
+
+Create a .profile file if missing
+
+```
+touch .profile
+```
+
+Edit the .profile file to insert:
+
+```
+alias tnbitpay='/Applications/Bitcoin-Qt.app/Contents/MacOS/Bitcoin-Qt -datadir=/Users/username/blockchains/bitcoin-core/networks/mainnet/'
+
+alias tnbitpayreg='/Applications/Bitcoin-Qt.app/Contents/MacOS/Bitcoin-Qt -datadir=/Users/username/blockchains/bitcoin-core/networks/regtest/'
```

Comment:
let's name the aliases <chain><network>

So bitcoinregtest, bitcointestnet, bitcoinmainnet, etc